release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -76,6 +76,12 @@ To install a CSI driver on a cluster running on vSphere, you must have the follo
 
 These components are still supported, but are deprecated; support for them will be removed in a future version of {product-title}.
 
+[id="ocp-4-10-installation-and-upgrade-alibaba-ipi"]
+==== Installing a cluster on Alibaba Cloud using installer-provisioned infrastructure
+
+{product-title} {product-version} introduces support for installing a cluster on Alibaba Cloud using installer-provisioned infrastructure. This type of installation lets you use the installation program to deploy a cluster on infrastructure that the installation program provisions and the cluster maintains.
```

Comment:
s/introduces support/introduces the ability to install a cluster on Alibaba Cloud using installer-provisioned infrastructure in Technology Preview.